.circleci/config.yml

```diff
@@ -1124,7 +1124,11 @@ jobs:
             build docs
 
             echo "Deploying docs"
-            docs/deploy_netlify.sh $CIRCLE_BRANCH $CIRCLE_PULL_REQUEST
+            if [ -n "$CIRCLE_PULL_REQUEST" ]; then
```

Comment:
The issue is that we're using CIRCLE prefixed variables, rather than the variables set in `setup_env`.
Just use `$BRANCH` and `$PULL_REQUEST` and they should be defined.

Reply:
Oh I see they're defined in a previous step, I'll change it